style/sass/README.md

```diff
@@ -38,8 +38,8 @@ Selectors
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid using the direct descendant selector `>`.
-* Avoid nesting more than 4 selectors deep.
-* Don't nest more than 6 selectors deep.
+* Avoid nesting more than 3 selectors deep.
```

Comment:
Personally, I *try* to keep it to no more than 1 level of nesting, but I do break that rule sometimes. To me, I feel that less nesting means stronger class names, more searchable class names, less complexity, and overall more thoughtful use of css. This makes me feel like the quality of my pattern goes up, and my code is *easier* to read. If we are making a style suggestion, I can get behind `Avoid nesting more than 3 selectors deep`, but I would prefer even `2 selectors deep`.